requirements/edx-sandbox/py38.txt

```diff
@@ -32,10 +38,8 @@ markupsafe==2.1.3
     # via
     #   chem
     #   openedx-calc
-matplotlib==3.3.4
-    # via
-    #   -c requirements/edx-sandbox/../constraints.txt
-    #   -r requirements/edx-sandbox/py38.in
+matplotlib==3.7.2
```

Comment:
There is no breaking change mentioned in the release notes of the `matplotlib` package in all the bumped versions so we should be good to go with this package upgrade without any issue.